### PR TITLE
Running multiple versions of mongodb discovery tests

### PR DESCRIPTION
**Pull request by samiura**
**Description:** <Describe what has changed.>

**Link to Splunk idea:** <Link to Splunk idea, see https://ideas.splunk.com>

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

File: Makefile
Comment (jeffreyc-splunk) [inline Makefile:84]: Should this change be reverted?

**Comment by samiura** [comment]
@jeffreyc-splunk I think we will have MR conflicts with your PRs which I have approved. I will wait for you to merge first.

**Comment by jeffreyc-splunk** [comment]
> @jeffreyc-splunk I think we will have MR conflicts with your PRs which I have approved. I will wait for you to merge first.

@samiura Please merge this first when you are ready.  I can then update my PR to support this new job.

**Review by atoulme — APPROVED** [review]

**Comment by atoulme** [comment]
These are only test changes ; tests pass ; approving.

**Pull request merged**
